spec/fizzbuzz_spec.rb

```diff
@@ -0,0 +1,36 @@
+require './solver'
+
+describe Solver do
+  context 'When testing the Solver class' do
+    solver = Solver.new
+
+    it 'the method factorial should return the factorial of given integer' do
+      value = 5
+      result = solver.factorial(value)
+      expect(result).to eq 120
+    end
+
+    it 'the method factorial should return the factorial of given integer' do
+      value = 4
+      result = solver.factorial(value)
+      expect(result).to eq 24
+    end
+
```

Comment:
- [ ] The name of this file is fizzbuzz but you are currently testing the factorial method. This means you have two factorial tests and no fizzbuzz test. Kindly make changes to the file and test the fizzbuzz method as stated in the requirements below

<img width="1080" alt="Screenshot 2022-03-11 at 08 58 55" src="https://user-images.githubusercontent.com/64774936/157826730-aa719441-9fdf-44e3-81d7-0b703d39e258.png">